### ISSUE TITLE
Error: command 'eslint.*' not found

### ISSUE THREAD

**Issue by TuxujPes** [Microsoft/vscode-eslint#252]
Hi there,
some time ago eslint has totally stopped working for me :(
And currently, I am out of ideas why.

tool/env | version
------------ | -------------
System | OS X El Capitan 10.11.6
VSCode | 1.13.1
ESLint plugin | 1.2.11
ESlint | 3.19.0

List of my extensions (not all of them enabled per workspace):
```
EditorConfig.EditorConfig
HookyQR.beautify
PKief.material-icon-theme
Tyriar.sort-lines
akamud.vscode-theme-onedark
alefragnani.project-manager
christian-kohler.npm-intellisense
christian-kohler.path-intellisense
dbaeumer.jshint
dbaeumer.vscode-eslint
deerawan.vscode-jasmine-es5
dzannotti.vscode-babel-coloring
eg2.tslint
freebroccolo.sml
gerane.Theme-Clouds
ionutvmi.spacegray-vscode
joelday.docthis
johnpapa.Angular2
ms-vscode.Theme-MaterialKit
ms-vscode.jscs
ms-vscode.sublime-keybindings
msjsdiag.debugger-for-chrome
redhat.java
robertohuertasm.vscode-icons
robinbentley.sass-indented
shinnn.stylelint
zhuangtongfa.Material-theme
```

Eslint commands are not found, nor eslint is listed in output's tool selection.
Where can I start to look for the root cause or to fix the issue?
Thanks

**Comment by TuxujPes** [comment]
bumping up. Any thoughts?

**Comment by dbaeumer** [comment]
Can you share a workspace (best via GitHub) that demos the behavior.

Since no ESLint commands are available (F1 > ESlint) it is very likely that the extension is disabled. The commands don't require to run any code. The come from the package.json file.

Another idea is that the ESLint installation is broken for some reason. Yoo could go to our home directory and then to .vscode/extensions and delete dbaeumer.vscode-eslint-1.2.11 and reinstall eslint.

**Comment by TuxujPes** [comment]
Removing folder with eslint in `.vscode/extensions` and reinstalling have helped indeed!
Should have tried that firsthand...
Thanks, really helped me! :)

**Comment by kvinlambert** [comment]
I'm having the same issue, tried pretty much everything, deleted all my extensions and downloaded only eslint, and nothing changed. I'm still have the "command 'eslint.*' not found" notification as soon as I try to enable the extension (which doesn't work at all for me) manually.

Any help would be greatly appreciated! 👍

**Comment by dbaeumer** [comment]
@kvinlambert if you open the developer tools (Help > Toogle Developer Tools) is there anything in the console ?

**Comment by Phuhwei** [comment]
@dbaeumer Same issue here with @kvinlambert, just happen today. Tried uninstall and reinstall.
console.ts:123 [Extension Host] Activating extension `dbaeumer.vscode-eslint` failed: Cannot find module '/Users/Len/.vscode/extensions/dbaeumer.vscode-eslint-1.4.1/client/out/extension'.
t.log @ console.ts:123
console.ts:123 [Extension Host] Activating extension `dbaeumer.vscode-eslint` failed:  Cannot find module '/Users/Len/.vscode/extensions/dbaeumer.vscode-eslint-1.4.1/client/out/extension'
t.log @ console.ts:123
console.ts:123 [Extension Host] Here is the error stack:  Error: Cannot find module '/Users/Len/.vscode/extensions/dbaeumer.vscode-eslint-1.4.1/client/out/extension'
	at Function.Module._resolveFilename (module.js:470:15)
	at Function.Module._load (module.js:418:25)
	at Function.X.o._load (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:28:422043)
	at Module.require (module.js:498:17)
	at require (internal/module.js:20:19)
	at Function.d [as __$__nodeRequire] (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js:4:27881)
	at m (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:28:437615)
	at e._doActivateExtension (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:28:442863)
	at e._activateExtension (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:28:442339)
	at Object.actualActivateExtension (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:28:440525)
extensionService.ts:433 Activating extension `dbaeumer.vscode-eslint` failed: Cannot find module '/Users/Len/.vscode/extensions/dbaeumer.vscode-eslint-1.4.1/client/out/extension'.
e._logMessageInConsole @ extensionService.ts:433
messageService.ts:125 command 'eslint.executeAutofix' not found

**Comment by dbaeumer** [comment]
This was me breaking something in 1.4.1. I published 1.4.2 which addresses this.

**Comment by bac-joker** [comment]
The computer power off suddenly shut down. Deleted  and downloaded eslint, and nothing changed. Reinstall vscode fixed this issue.
